test/Streamly/Test/Data/Fold.hs

```diff
@@ -1,15 +1,26 @@
 module Main (main) where
 
-import Prelude hiding (maximum, minimum, elem, notElem, null, product, sum, head, last)
-import qualified Streamly.Internal.Data.Fold as F
-import qualified Streamly.Prelude as S
-import qualified Streamly.Data.Fold as FL
-
+import Data.Semigroup as SG
```

Comment:
When we are importing a module `as xyz` then we should always import it `qualified`. However, the semigroup module should not be imported qualified we can just import symbols from it as `import Data.Semigroup (x, y, z).